WordPress/src/main/java/org/wordpress/android/util/URLFilteredWebViewClient.java

```diff
@@ -35,10 +45,37 @@ public URLFilteredWebViewClient(Collection<String> urls) {
         mAllowedURLs.addAll(urls);
     }
 
+    public void setListener(URLWebViewClientListener listener) {
+        mListener = listener;
+    }
+
     protected boolean isAllURLsAllowed() {
         return mAllowedURLs.size() == 0;
     }
 
+    @Override
+    public void onPageStarted(WebView view, String url, Bitmap favicon) {
```

Comment:
What are our options to factorize this code? We have the same logic in `URLFilteredWebViewClient` and `JetpackConnectionWebViewClient`, this can be a source of problems if someone change one file but miss the other.

Reply:
Instead of having those 2 classes extend from `WebViewClient` create another class that extends `WebViewClient` and move all that shared logic into there, and then have `URLFilteredWebViewClient` and `JetpackConnectionWebViewClient` extend that new class

Reply:
Yes, I think we should write this intermediate class.